wazp/callbacks.py

```diff
@@ -259,3 +314,122 @@ def modify_rows_selection(
             alert_state,
             alert_message,
         )
+
+
+def get_dashboard_callbacks(app):
+    @app.callback(
+        Output("table-container", "children"),
+        Input("table-container", "children"),
+        State("session-storage", "data"),
+    )
+    def create_input_data_tbl(
+        tbl_container_children: list,
+        cfg_params_in_storage: tuple,
```

Comment:
```suggestion
        table_container_children: list,
        config_params_in_storage: tuple,
```
I mean, config is already an abbreviation. You want to abbreviate the abbreviation?!
I guess I'm ok with "params".